vi/cheatsheet-deep-learning.md

```diff
@@ -0,0 +1,321 @@
+
+<br>
+
+**31. Definitions**
+
+&#10230; Định nghĩa
+
+<br>
+
+**32. Markov decision processes ― A Markov decision process (MDP) is a 5-tuple (S,A,{Psa},γ,R) where:**
+
+&#10230; Tiến trình quyết định Markov (Markov decision processes) - Tiến trình quyết định Markov (MDP) là một dạng 5-tuple (S,A,{Psa},γ,R) mà ở đó:
+
+<br>
+
+**33. S is the set of states**
+
+&#10230; S là tập hợp các trạng thái (states)
+
+<br>
+
+**34. A is the set of actions**
+
+&#10230; A là tập hợp các hành động (actions)
+
+<br>
+
+**35. {Psa} are the state transition probabilities for s∈S and a∈A**
+
+&#10230; {Psa} là xác suất chuyển tiếp trạng thái cho s∈S và a∈A
+
+<br>
+
+**36. γ∈[0,1[ is the discount factor**
+
+&#10230; γ∈[0,1[ là discount factor
+
+<br>
+
+**37. R:S×A⟶R or R:S⟶R is the reward function that the algorithm wants to maximize**
+
+&#10230; R:S×A⟶R hoặc R:S⟶R là reward function (hàm định nghĩa phần thưởng) mà giải thuật muốn tối đa hoá.
+
+<br>
+
+**38. Policy ― A policy π is a function π:S⟶A that maps states to actions.**
+
+&#10230; Policy - Policy π là 1 hàm π:S⟶A có nhiệm vụ ánh xạ states tới actions
+
+<br>
+
+**39. Remark: we say that we execute a given policy π if given a state s we take the action a=π(s).**
+
+&#10230; Chú ý: Ta quy ước rằng ta thực thi policy π cho trước nếu cho trước state s ta có action a=π(s)
+
+<br>
+
+**40. Value function ― For a given policy π and a given state s, we define the value function Vπ as follows:**
+
+&#10230; Hàm giá trị (Value function) - Với policy cho trước π và state s, ta định nghĩa value function Vπ như sau:
+
+<br>
+
+**41. Bellman equation ― The optimal Bellman equations characterizes the value function Vπ∗ of the optimal policy π∗:**
+
+&#10230; Phương trình Bellman - Phương trình tối ưu Bellman đặc trưng hoá value function Vπ∗ của policy tối ưu (optimal policy) π∗:
+
+<br>
+
+**42. Remark: we note that the optimal policy π∗ for a given state s is such that:**
+
+&#10230; Chú ý: ta quy ước optimal policy π∗ đối với state s cho trước như sau:
+
+<br>
+
+**43. Value iteration algorithm ― The value iteration algorithm is in two steps:**
+
+&#10230; Giải thuật duyệt giá trị (Value iteration) - Giải thuật duyệt giá trị gồm 2 bước:
+
+<br>
+
+**44. 1) We initialize the value:**
+
+&#10230; 1) Ta khởi tạo giá trị (value):
+
+<br>
+
+**45. 2) We iterate the value based on the values before:**
+
+&#10230; 2) Ta duyệt qua giá trị dựa theo giá trị phía trước:
+
+<br>
+
+**46. Maximum likelihood estimate ― The maximum likelihood estimates for the state transition probabilities are as follows:**
+
+&#10230; Ước lượng khả năng tối đa (Maximum likelihood estimate) - Ước lượng khả năng tối đa cho xác suất chuyển tiếp trạng thái (state) sẽ như sau:
+
+<br>
+
+**47. times took action a in state s and got to s′**
+
+&#10230; thời gian hành động a tiêu tốn cho state s và biến đổi nó thành s′
+
+<br>
+
+**48. times took action a in state s**
+
+&#10230; thời gian hành động a tiêu tốn cho state (trạng thái) s
```

Comment:
```suggestion
&#10230; số lần hành động a tiêu tốn cho state (trạng thái) s
```

Reply:
I think to keep it in **thời gian** is more suitable because we have **tiêu tốn** in this sentence.